src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -155,6 +155,22 @@ class HMICapabilitiesTest : public ::testing::Test {
         .WillByDefault(ReturnRef(kHmiCapabilitiesCacheFile));
 
     hmi_capabilities_ = std::make_shared<HMICapabilitiesImpl>(mock_app_mngr_);
+    IsReadyResponsesReceived();
+  }
+
+  void IsReadyResponsesReceived(){
+    hmi_capabilities_->UpdateRequestsRequiredForCapabilities(
+        hmi_apis::FunctionID::RC_IsReady);
+    hmi_capabilities_->UpdateRequestsRequiredForCapabilities(
+        hmi_apis::FunctionID::VR_IsReady);
+    hmi_capabilities_->UpdateRequestsRequiredForCapabilities(
+        hmi_apis::FunctionID::UI_IsReady);
+    hmi_capabilities_->UpdateRequestsRequiredForCapabilities(
+        hmi_apis::FunctionID::TTS_IsReady);
+    hmi_capabilities_->UpdateRequestsRequiredForCapabilities(
+        hmi_apis::FunctionID::Navigation_IsReady);
+    hmi_capabilities_->UpdateRequestsRequiredForCapabilities(
+        hmi_apis::FunctionID::VehicleInfo_IsReady);
```

Comment:
@LitvinenkoIra I think you can create a const vector like for `light_name_enum_values` and just use it in SetUp:

```
  for (const auto& request : requests) {
      hmi_capabilities_->UpdateRequestsRequiredForCapabilities(request);    
    }
```

Reply:
Done https://github.com/LuxoftSDL/sdl_core/pull/179/commits/9b151783d89e8a8b917616ac5d11c0fc05db2887#diff-33052932a1a43a40974e7501d25c37b48ab1aa8a8b0e29ee4efc5995b8eb0096R133